### PR TITLE
Make Gloss Detail panels clickable across heading

### PR DESCRIPTION
**Pull request by susanodd**
as is done for Search Panels. Removed excess divs. Problems with Django if-else-endif mismatches so some minor adjustments.

### CODE REVIEW

**Comment by susanodd** [comment]
Need to so for morphemes too. (As a separate commit.)

**Comment by susanodd** [comment]
Morphemes are done.

File: signbank/dictionary/templates/dictionary/gloss_detail.html [hunk 1]
```diff
@@ -171,6 +171,8 @@
 </script>
 
     <script type='text/javascript' src="{{STATIC_URL}}js/gloss_edit_color.js"></script>
+    <script type="text/javascript" src="{{STATIC_URL}}js/drag_drop_files.js"></script>
+    <script type="text/javascript" src="{{STATIC_URL}}js/recordscript.js"></script>
```
Comment (vanlummelhuizen) [inline signbank/dictionary/templates/dictionary/gloss_detail.html:175]: @susanodd Could you point out where these Javascripts are used?

Comment (susanodd) [reply signbank/dictionary/templates/dictionary/gloss_detail.html:175]: Oh, I replied below. They are used in the record video routines. They were inside of a permission-prefaced block with lots of nested blocks inside it.

**Comment by susanodd** [comment]
I removed one of these:

   {% if perms.dictionary.view_advanced_properties %}

from gloss_detail.html
That's the one that wasn't parsing.
There are way too many perms things inside of GlossDetailView. It's very difficult for a human to parse.

**Comment by susanodd** [comment]
@vanlummelhuizen do you agree this can be merged?

**Comment by susanodd** [comment]
Nice idea!!

**Comment by vanlummelhuizen** [comment]
@susanodd Nice alternative for changing the mouse cursor. I think this can be merged.

**Pull request merged**